interface/wx/memoryallocator.h

```diff
@@ -0,0 +1,98 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        memoryallocator.h
+// Purpose:     interface of wxMemoryAllocator
+// Author:      wxWidgets team
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+/**
+    @class wxMemoryAllocator
+
+    This is a little helper class that hides the difference between
+    C++ operators new/delete and C way of memory management using
+    malloc()/free(). It is currently being used in wxImage to do
+    memory management. Right now it has 3 direct descendants - 1
```

Comment:
All things considered, I believe removing the sentence starting with "Right now" would actually improve the description.